README.md

```diff
@@ -574,13 +626,13 @@ print(f"Prediction: {output}")
 ### C++
 
 One common pattern is to use SmartSim to spin up the Orchestrator database
-and then use the Python client to set the model in
-the database. Once set, an application that uses
-the C, C++, or Fortran clients will call the model that was set.
+and then use the Python client to set the model in the database. Once set, an
+application written in C, C++, or Fortran will utilize their respective client
+to call the model that was set and retrieve the results as a language native tensor.
 
-This example shows the necessary code an application would need to include
-to execute a model (with any ML backend) that had been stored prior to application
-launch by the Python client.
+This example shows how, with minimal boilerplate code, a C++ application launched from
+SmartSim is able utilize the Client API to execute a model stored in the Orchestrator
```

Comment:
"is able utilize" -> "is able to utilize"

Reply:
Fixed at https://github.com/CrayLabs/SmartSim/pull/136/commits/d2b3489b119a3308afa09974954c57eb9eb9ba3c